torchdata/datapipes/iter/transform/bucketbatcher.py

```diff
@@ -204,13 +204,25 @@ class MaxTokenBucketizerIterDataPipe(IterDataPipe[DataChunk[T_co]]):
     For an example in the audio domain, it may be batching samples with similar length. Then, given the
     ``max_token_count``, each batch may be concatenated to a Tensor with the same size and minimum padding.
 
+    If ``include_padding`` is set to ``True``, the token count of each batch includes the padding a succeeding
+    DataPipe could add. This guarentees that even after the batch is padded, ``max_token_count`` will not be exceeded.
+    This can prevent out-of-memory issues for data with large variations in length.
+
+    Note that batches are bucketized starting from the smallest size in a buffer.
+    This can limit the variablity of batches if ``buffer_size`` is large.
+    To increase variablity, apply ``torchdata.datapipes.iter.Shuffler`` before this DataPipe,
+    and keep ``buffer_size`` small.
+
+
     Args:
         datapipe: Iterable DataPipe being batched
         max_token_count: Maximum length of total length of data in each batch
         len_fn: Function to be applied to each element to get lengths. ``len(data)`` is used by default.
         min_len: Optional minimum length to be included into each batch
         max_len: Optional maximum length to be included into each batch.
         buffer_size: This restricts how many tokens are taken from prior DataPipe to bucketize
+        include_padding: If true, assume data will be padded to the largest length in batch in succeeding DataPipes.
```

Comment:
This doesn't reflect the fact that this function will compare the padded length with the `max_token_count`.